src/main/InternalEnforcer.lua

```diff
@@ -371,6 +371,50 @@ function InternalEnforcer:removeFilteredPolicy(sec, ptype, fieldIndex, fieldValu
     return true
 end
 
+function InternalEnforcer:updateFilteredPolicies(sec, ptype, newRules, fieldIndex, fieldValues)
+    if fieldValues == nil or #fieldValues == 0 then
+        return false
+    end
+
+    local oldRules={}
+    if self:shouldPersist() then
+
+        local status, err = pcall(function () self.adapter:updateFilteredPolicies(sec, ptype, newRules, fieldIndex, fieldValues) end)
```

Comment:
Please also create a `updateFilteredPolicies` function in FileAdapter too with a `not implemented` error like other such functions, otherwise there is no way to try this out.

Reply:
OK,I’ll fix it.